### PR TITLE
added hello project

### PR DESCRIPTION
**Pull request by tonnytech**
Did the following in this branch: 

- Create a new Rails app called 'hello-rails-react' by using the command rails new -j webpack -d=postgresql hello-rails-react
- Create a babel.config.js in the root folder of the project.
- Update webpack.config.js to include the babel-loader.
- Make sure that the Postgres database is set up.
- Create a table for storing messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table.
- Create a static view that will be the root of your app.
- Create the App component with react-router and render it in the static view.
- Create the Greeting component that will display a greeting. Set it up as a route in the App component.
- Create a store, an action, and a reducer that will connect to the API endpoint to get the random greeting.
- Display the random greeting in the Greeting component.

### CODE REVIEW

File: app/controllers/greetings_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class GreetingsController < ApplicationController
   def random_greeting
-    message = Message.order("RANDOM()").first
+    message = Message.order('RANDOM()').first
     render json: { greeting: message&.content }
   end
```
Comment (Shedrack-Sunday) [inline app/controllers/greetings_controller.rb:5]: -[ ] In the `seed.rb` file, you're meant to create `5` different greetings which would be used to test your API endpoint to your front end. Kindly so so 👍 

### Requirement:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @tonnytech  👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with Rails model✔️
- Kudus making the requirement asked      ✔️  

# - Required Changes  🌵  

- [ ] Kindly take note to add the `5` suggested greetings in the seed file, so your code executes a random greeting everytime 👍 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/controllers/greetings_controller.rb (hunk 1 above)
Comment (tonnytech) [reply app/controllers/greetings_controller.rb:5]: Hi @Shedrack-Sunday,

Thanks for the feedback. However according to the instructions, I'm required to display one random greeting at a time. Check this out:
1. I'm required to create an API endpoint that selects A RANDOM GREETING
firstGreeting

2. I'm required to create a Greeting component that will display a greeting:

secondG Reeting

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @tonnytech,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**